### PR TITLE
migrate ProductCard component to tailwind

### PR DESCRIPTION
**Pull request by yashtotla**
Related issue – https://github.com/ethereum/ethereum-org-website/issues/13946

### CODE REVIEW

File: src/components/ProductCard.tsx [hunk 1]
```diff
@@ -87,71 +82,53 @@ const ProductCard = ({
...
-      border="1px"
-      borderColor="lightBorder"
-      textDecoration="none"
-      _hover={{
-        transition: "transform 0.1s",
-        transform: "scale(1.02)",
-      }}
+      className={cn(
+        "text-text bg-searchBackground flex-col justify-between",
+        "shadow-product-card",
```
Comment (pettinarip) [inline src/components/ProductCard.tsx:90]: @nloureiro this card has a few custom shadows, Im thinking that perhaps we can simply remove them or replace them by one of the default TW shadows like `shadow-md` or `shadow-lg`. LMK what do you think.
You can see these cards used in https://ethereum.org/en/developers/local-environment/

File: src/components/ProductCard.tsx [hunk 2]
```diff
@@ -87,71 +82,53 @@ const ProductCard = ({
...
-        background={bgProp}
-        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
-        minH="200px"
+        className="min-h-[200px] shadow-product-card-img-container"
+        style={{ backgroundColor: bgProp }}
       >
-        <Image
-          src={image}
-          alt={alt}
-          height="100"
```
Comment (pettinarip) [inline src/components/ProductCard.tsx:121]: We still need this to tell next image the size of the image.

**Review by pettinarip — CHANGES_REQUESTED** [review]
@yashtotla thanks for the PR.

Left a few comments. I've also tagged @nloureiro for design approval since there are a few old colors and shadows that would be nice if we can replace them and match our new DS styles.

File: src/components/ProductCard.tsx (hunk 1 above)
Comment (yashtotla) [reply src/components/ProductCard.tsx:89]: I will update the box shadow styles once I have more clarity. I have addressed the rest of your comments.

**Review by pettinarip — APPROVED** [review]
Thanks @yashtotla

**Pull request merged**